README.md

```diff
@@ -34,14 +36,29 @@ There is a file containing the example calls. Run:
 $ python ./src/code_analyzer/example.py <path-to-your-repo>
 ```
 
+If `test_creation` assists your research, please cite it as follows:
+```bibtex
+@inproceedings{DBLP:conf/recsys/Kula15,
```

Comment:
I don't think we should use `@inproceedings` as we are not working on publishing a paper. `@software`, `@manual`, or `@misc` would be more appropriate.

Also, GitHub does support a separate [CITATION.cff](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files) in case that we are to provide citations. Maybe we should put the citation information there instead.

Reply:
Ah, yes, I found that image is not cute, so I removed it. Will add one later if I can find a suitable one.